test_util/cluster.py

```diff
@@ -381,15 +381,13 @@ def master_upgrade_order(cluster):
         )]
 
     def upgrade_host(tunnel, role, bootstrap_url):
-        # Download the install script for the new DC/OS.
-        tunnel.remote_cmd(curl_cmd + ['--remote-name', bootstrap_url + '/dcos_install.sh'])
+        # Download the upgrade script for the new DC/OS.
+        tunnel.remote_cmd(curl_cmd + ['--remote-name', bootstrap_url + 'upgrade.latest'])
+        tunnel.remote_cmd(curl_cmd + ['--remote-name', bootstrap_url + '/upgrade/' +
+                                      "'`cat upgrade.latest`'" + '/dcos_node_upgrade.sh'])
 
-        # Remove the old DC/OS.
-        tunnel.remote_cmd(['sudo', '-i', '/opt/mesosphere/bin/pkgpanda', 'uninstall'])
-        tunnel.remote_cmd(['sudo', 'rm', '-rf', '/opt/mesosphere', '/etc/mesosphere'])
-
-        # Install the new DC/OS.
-        tunnel.remote_cmd(['sudo', 'bash', 'dcos_install.sh', '-d', role])
+        # Upgrade to the new DC/OS.
+        tunnel.remote_cmd(['sudo', 'bash', 'dcos_node_upgrade.sh', '-d', role])
```

Comment:
ah good point, i will need to pass the args, i changed the code so it takes args.